src/components/_common/FormFields/Input/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useField } from 'formik';
+import cn from 'classnames';
+
+import ErrorMessage from 'components/_common/ErrorMessage';
+import styles from './styles.module.scss';
+
+type InputProps = {
+  label: string;
+  name: string;
+  className?: string;
+} & React.InputHTMLAttributes<HTMLInputElement>;
+
+const Input: React.FC<InputProps> = ({ label, ...props }) => {
+  const [showPassword, setShowPassword] = React.useState(false);
+  const [field, meta] = useField({ ...props });
+  const isPasswordType = props.type === 'password';
```

Comment:
Can we use destructuring from `props` for `type` too?

Reply:
make sense, will fix in the next task